browser/brave_webcompat_reporter/webcompat_reporter_service.cc

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
the filename should match the class name. This should be `browser/webcompat_reporter/webcompat_reporter_service` with class name `WebcompatReporterService`. We only add `Brave*` when we're overriding things from Chromium